.github/workflows/docker.yml

```diff
@@ -56,10 +49,11 @@ jobs:
           tags: infrahelpers/cloud-python:pyspark-emr-6-light
           cache-from: type=gha
           cache-to: type=gha,mode=max
+          platforms: linux/amd64,linux/arm64
```

Comment:
Equivalent to this
```bash
cd pyspark-emr-6-coretto-8-light
docker buildx build --platform linux/amd64,linux/arm64 -t infrahelpers/cloud-python:pyspark-emr-6-light --push .
```

See https://docs.docker.com/build/buildx/multiplatform-images/ for more information